README.md

```diff
@@ -17,36 +17,28 @@ For example, it takes one command to get a REST API, a GraphQL API or a fully op
 npm install -g lumber-cli -s
 ```
 
-## Plugins
-
-|Plugin                                                                  | Description                                                                                |
-|------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
-|[lumber-forestadmin](https://github.com/ForestAdmin/lumber-forestadmin) | Install Forest Admin to get an off-the-shelf admin panel based on a highly-extensible API. |
-|[lumber-graphql](https://github.com/ForestAdmin/lumber-graphql)         | Generate a GraphQL API.                                                                    |
-|[lumber-jwt](https://github.com/ForestAdmin/lumber-jwt)                 | Add a JWT authentication middleware.                                                       |
-
 ## See it in action
 
 ```
 $ npm install -g lumber-cli -s
-$ lumber generate -c postgres://erlich:aviato@localhost:5432/meals Meals
+$ lumber generate -c "postgres://erlich:aviato@localhost:5432/meals" Meals
 
 $ cd Meals/ && npm install -s
 $ npm start
 
 Your application is listening on port 3310.
 ```
 
-Finally, visit http://localhost:3310.
+Finally, visit https://app.forestadmin.com/.
 
 ## Commands
 
 `$ lumber [command]`
 
 - `generate <appName>`      generate a backend application with an ORM/ODM configured.
-- `install <package>`       install a Lumber plugin.
-- `run <plugin:cmd>` run a command from a Lumber plugin.
-- `help [cmd]`    display help for [cmd]
+- `login`                   log in on forestadmin api.
```

Comment:
line 14 --> log into forestadmin api (FYI https://grammarist.com/spelling/log-in-login/ so it should be "log in" not "login"; however, here "in" becomes "into" because there is a notion of movement forward, therefore "log into").